examples/gno.land/r/demo/foo20/foo20.gno

```diff
@@ -1,112 +1,143 @@
 package foo20
 
 import (
+	"math"
 	"std"
 	"strings"
 
 	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/ownable"
 	"gno.land/p/demo/ufmt"
 	"gno.land/r/demo/users"
 
 	pusers "gno.land/p/demo/users"
 )
 
 var (
-	foo   *grc20.AdminToken
-	admin std.Address = "g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj" // TODO: helper to change admin
+	foo      *grc20.Token
+	o        *ownable.Ownable
+	fauceted bool
 )
 
 func init() {
-	foo = grc20.NewAdminToken("Foo", "FOO", 4)
-	foo.Mint(admin, 1000000*10000)                                    // @administrator (1M)
-	foo.Mint("g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq", 10000*10000) // @manfred (10k)
+	// Instantiate ownable
+	o = ownable.New()
+	o.TransferOwnership("g125em6arxsnj49vx35f0n0z34putv5ty3376fg5") // @admin
```

Comment:
Let's remove the token owner completely and expose a "Faucet" function for the demonstration.

See #2388 for an example.